framework/components/ecl-social-media-links/ecl-social-media-links.scss

```diff
@@ -0,0 +1,45 @@
+/**
+ * Social media
+ * @define social-media-link
+ */
+
+.ecl-social-media-link {
+  @include ecl-hidden-print();
+
+  background-color: map-get($ecl-colors, 'grey-10');
+  display: block;
+  margin: 0;
+  padding: map-get($ecl-spacing, 'm');
+  width: 100%;
+}
+
+.ecl-social-media-link__item {
+  margin-bottom: map-get($ecl-spacing, 'xxs');
+  vertical-align: top;
+
+  &:last-child {
+    margin-bottom: 0;
+  }
+}
+
+.ecl-social-media-link__link {
+  align-items: center;
+  display: flex;
+  min-height: 2rem;
```

Comment:
I'm not a huge fan of hardcoded values or `min-height`s, but I'll accept this one